config/locales/en.yml

```diff
@@ -26,3 +26,21 @@ en:
         notice: "You have been logged out."
   authorization:
     unauthorized: "You are not authorized to access this page."
+  date_time:
+    distance_in_words:
+      short:
+        less_than_x_minutes:
+          one: "now"
+          other: "%{count}m"
+        x_minutes: "%{count}m"
+        about_x_hours:
+          one: "1hr"
+          other: "%{count}hrs"
+        x_days:
+          one: "1day"
+          other: "%{count}days"
+        about_x_months:
+          one: "1month"
+          other: "%{count}months"
+        x_months: "%{count}months"
```

Comment:
See the [Action View Date Helpers](https://api.rubyonrails.org/v7.1.3.4/classes/ActionView/Helpers/DateHelper.html#method-i-distance_of_time_in_words) for context